lib/livebook_web/live/session_live/app_teams_live.ex

```diff
@@ -66,8 +66,15 @@ defmodule LivebookWeb.SessionLive.AppTeamsLive do
       </div>
 
       <div :if={@messages != []} class="flex flex-col gap-2">
-        <.message_box :for={{kind, message} <- @messages} kind={kind}>
-          <%= raw(message) %>
+        <.message_box :for={{kind, message} <- @messages} message={message} kind={kind}>
+          <.link
+            :if={kind == :success}
+            href={"#{Livebook.Config.teams_url()}/orgs/#{@hub.org_id}"}
+            target="_blank"
+            class="font-medium text-blue-600"
+          >
+            See all deployed apps
```

Comment:
```suggestion
            <span>See all deployed apps</span>
            <.remix_icon icon="arrow-right-line" />
```

Reply:
Since it will open an external page, shouldn't we use `external-link-line`? 

![image](https://github.com/livebook-dev/livebook/assets/6402997/526eee39-62c0-4cca-850c-9cdd94a20125)

Reply:
I like the idea. 👍

Reply:
Sounds good to me!